### DISCUSSION TITLE
deploying mojo apps "efficiently" in a shared hosting / cPanel environments

### DISCUSSION THREAD

**Discussion by rmallah** [mojolicious/mojo#1932] [Q&A]
Hi 

I was toying with the idea of easy deployment of mojo apps in popular web hosting
environments like cPanel or ispconfig3.

Can hypnotoad be launched from a cronjob in cPanel environment and keep a the app persistent ?
also how can requests to particular urls be mapped to app server listening on some non-privileged port
considering the fact that ProxyPass may not be allowed in .htaccess and "normal" users do  not have
access to server configs.

any ideas?

**Comment by danimera** [comment]
that is crazy,, i did try that but does not work,, just permission issues, but you can run in cgi mode
